utils/models.py

```diff
@@ -100,9 +100,10 @@ def get_model(self, model_name: str, extra_args: dict=None, to_compile: bool=Tru
         model.eval()
 
         if to_compile:
-            match = re.match(r'([0-9]+\.[0-9]+)', torch.__version__)
-            if match and float(match.group(1)) >= 2.1:  # PyTorch will not support our model before v2.1
-                model = torch.compile(model)
+            # match = re.match(r'([0-9]+\.[0-9]+)', torch.__version__)
+            # if match and float(match.group(1)) >= 2.1:  # PyTorch will not support our model before v2.1
+            #     model = torch.compile(model)
+            pass  # Still unsupported :')
```

Comment:
either add another if/flag to the run command, or remove entirely.

Reply:
I prefer keeping it so that when in the future it will be supported we will be easily able to put it back in